### ISSUE TITLE
No pages will load

### ISSUE THREAD

**Issue by LoneFenris** [AllanWang/Frost-for-Facebook#1958]
- [X] I have verified that the bug does not occur in my mobile browser (at https://touch.facebook.com/)

**Describe the bug**
No pages seem to load in Frost anymore for me in the current (now legacy) version. Issue has been present the last few weeks. I tried clearing cache and storage to no effect. Clearing storage allowed me to successfully set up the account again, but once logged in nothing works.

I know the current release is deprecated as it gets rewritten, but I thought I'd at least document this. 

**To Reproduce**
Steps to reproduce the behaviour:
1. Open application
2. No page loads

**Details (please provide at least the app version):**
 - App Version: 3.2.0
 - Device: Xperia 5 III
 - Android Version: 13

**Logs/Screenshots:**

**Comment by pesasa** [comment]
I have the same problem. Just white page.

I have tried to clear cache and storage. Uninstalled and re-installed couple of times.

Weird thing is, that it works fine on my wife's phone. (Also OnePlus)

- App version: 3.2.0
- Device: OnePlus Nord 2T 5G
- Android version: 13
- Installed from F-Droid

**Comment by Bennett-69** [comment]
Same problem after similar troubleshooting here, broke suddenly a couple of weeks ago.

App version: 3.2.0
Device: OnePlus 7T
Android version: 13 (Paranoid Android Topaz 5)
Installed from F-Droid

**Comment by drakenabarion** [comment]
Same problem for me!
OnePlus Nord 2T 5g
Android 13 stock oxygenOS
Frost 3.2.0

**Comment by pasnox** [comment]
same issue, android 13 with OPPO find x3 lite

**Comment by LerouxGael** [comment]
Same issue on Android 13, Redmi note 10S

**Comment by gigachaand** [comment]
Same issue on Poco X5 5G Android 13

**Comment by mahashel** [comment]
Same issue. Launching authenticated app suddenly stopped loading pages or views about a month ago. No Home, messages, or even settings. Sadly, App is a brick. Tried all kinds of stuff: clearing cache, storage, reinstalling, reauth. No luck.
It's been a while now, hoping this was a transient backend issue that might resolve itself but looks like it's truly dead. I know the Dev has been gradually rewriting the app for years now. Looking forward to that release, but i can't wait on legacy app fix anymore. Will have to look elsewhere. Sucks, as i really liked Frost.

Asus Zenfone 8
Android 13 (July '23 security update), stock 
FB web loads fine in mobile browser

**Comment by trebuh** [comment]
Same issue here.
* Phone: OnePlus 3T
* LineageOS version: 18.1 (September update)
* Android version: 11
* Using microG

**Comment by Cwpute** [comment]
Same.
* Phone: Nokia TA-1157
* Android 11
* Frost version 3.2.0

**Comment by mormegil-cz** [comment]
Probably a duplicate of #1963
